.chant/prompts/verify.md

```
@@ -0,0 +1,59 @@
+---
+name: verify
+purpose: Verify that acceptance criteria are met
+---
+
+# Verify Acceptance Criteria
+
+You are verifying that a spec for {{project.name}} meets its acceptance criteria.
+
+## Your Spec
+
+**{{spec.title}}**
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)